tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/PacketTracer.cs

```diff
@@ -0,0 +1,261 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System.Runtime.Intrinsics.X86;
+using static System.Runtime.Intrinsics.X86.Avx;
+using static System.Runtime.Intrinsics.X86.Avx2;
+using static System.Runtime.Intrinsics.X86.Sse2;
+using System.Runtime.Intrinsics;
+using System;
+
+using ColorPacket256 = VectorPacket256;
+
+internal class Packet256Tracer
+{
+    public int Width { get; private set; }
+    public int Hight { get; private set; }
+    private static readonly int MaxDepth = 5;
+
+    public Packet256Tracer(int _width, int _hight)
+    {
+        if (_width % VectorPacket256.Packet256Size != 0)
+        {
+            _width += VectorPacket256.Packet256Size - _width % VectorPacket256.Packet256Size;
+        }
+        Width = _width;
+        Hight = _hight;
+    }
+
+    internal unsafe void RenderVectorized(Scene scene, int* rgb)
+    {
+        Camera camera = scene.Camera;
+        for (int y = 0; y < Hight; y++)
+        {
+            int stride = y * Width;
+            for (int x = 0; x < Width; x += VectorPacket256.Packet256Size)
+            {
+                float fx = (float)x;
+                Vector256<float> Xs = SetVector256(fx + 7, fx + 6, fx + 5, fx + 4, fx + 3, fx + 2, fx + 1, fx);
```

Comment:
It looks like this is simply `_mm256_add_ps(_mm256_set1_ps(fx), _mm256_set_ps(7,6,5,4,3,2,1,0))`

Reply:
Good point, will change.